setup.py

```diff
@@ -15,16 +15,45 @@
 # limitations under the License.
 
 # THIS FILE IS MANAGED BY THE GLOBAL REQUIREMENTS REPO - DO NOT EDIT
+import os
 import setuptools
```

Comment:
What about this hack:

https://github.com/kelp404/six/blob/master/setup.py#L3-L6